theano/gof/type.py

```diff
@@ -35,6 +35,19 @@ class CLinkerType(CLinkerObject):
 
     """
 
+    def c_element_type(self):
+        """
+        Optional: Return the name of the primitive C type of items into variables
+        handled by this type.
+
+        e.g:
+
+         - For ``TensorType(dtype='int64', ...)``: should return ``"npy_int64"``.
+         - For ``GpuArrayType(dtype='int32', ...)``: should return ``"ga_int"``.
+
+        """
+        return ''
```

Comment:
That was just to avoid using `try-catch` block when looking for `c_element_type()` in an op ^^ ! If it isn't overrided, then the returned string is empty and then evaluated to False.

Is it better to raise an exception ?